_sass/_hackathon.scss

```diff
@@ -0,0 +1,33 @@
+
+#csh-hacks-content{
+  padding-top: 10px;
+  padding-bottom: 10px;
+  background: rgba(0,0,0,0.0);
+  & h2{
+    color: white;
```

Comment:
You're setting the text color to something below but then overriding it with white here? Also, add spacing between `h2` and `{`

Reply:
I want the paragraph text to be rgba(217, 184, 206, 1.00) and the header color white so that I have a consistent style with hacks.csh.rit.edu